widgets/common/src/LabelsInput.jsx

```diff
@@ -102,12 +105,14 @@ export default function LabelsInput({ initialLabels, onChange, toolbox }) {
 }
 
 LabelsInput.propTypes = {
+    addMode: PropTypes.bool,
     initialLabels: LabelsPropType,
     onChange: PropTypes.func.isRequired,
     toolbox: Stage.PropTypes.Toolbox.isRequired
 };
 
 LabelsInput.defaultProps = {
+    addMode: false,
```

Comment:
The functionality sounds more like `appendMode` :smile:

Reply:
Or, to make it more descriptive, we could do

```suggestion
    appendToExistingMode: false,
```

Reply:
In the context of the widget - it's an append mode. But in the context of this particular component I wouldn't say so - appending means adding something to existing items, while in `addMode` behaviour of this components is actually shifted in an opposite direction - labels passed in `initialLabels` are actually being hidden from the view.

Reply:
Ok, I see your point. To me, _add mode_ is still cryptic, because we are adding labels anyway :slightly_smiling_face: Maybe we could settle on renaming it to something else, perhaps `hideInitialLabels`?

Reply:
Sounds good - changed.